dev/tidb-in-kubernetes/maintain/tidb-binlog.md

```diff
@@ -0,0 +1,116 @@
+---
+title: TiDB Binlog 运维
+summary: 了解如何在 Kubernetes 上运维 TiDB 集群的 TiDB Binlog。
+category: how-to
+---
+
+# TiDB Binlog 运维
+
+本文档介绍如何在 Kubernetes 上运维 TiDB 集群的 [TiDB Binlog](/dev/reference/tools/tidb-binlog/overview.md)。
+
+## 运维准备
+
+- [部署 TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md);
```

Comment:
```suggestion
- [部署 TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md)；
```

Shall we use CN semicolon?

Reply:
Yes. Comment addressed in [02cef83](https://github.com/pingcap/docs-cn/pull/1959/commits/02cef836d282f51fdfb371524b3676e3b0aca0a0).